src/feature/control/control_getinfo.c

```diff
@@ -622,16 +613,17 @@ getinfo_helper_dir(control_connection_t *control_conn,
     smartlist_free(descs);
   } else if (!strcmpstart(question, "dir/status/")) {
     *answer = tor_strdup("");
-  } else if (-1 == getinfo_helper_current_consensus(
-                                                    question,
-                                                    answer,
-                                                    errmsg))
-  {
-    /**
-     * answer is set by getinfo_helper_current_consensus
-     * if the question matches
-     */
-    return -1;
+  } else if (!strcmp(question, "dir/status-vote/current/consensus")) {
+    if (getinfo_helper_current_consensus(
```

Comment:
Please don't wrap the code like this. Check the rest of the code, we usually wrap it between the arguments and not after the parentheses.

We also usually do `< 0` when checking for errors and not `== -1`.